lib/cookie.js

```diff
@@ -620,6 +636,43 @@ Cookie.prototype.inspect = function inspect() {
     '"';
 };
 
+Cookie.prototype.toJSON = function() {
+  var obj = {};
+
+  var props = this.serializableProperties || Cookie.serializableProperties;
```

Comment:
You're not mistaken. I think I'll keep it simple and get rid of `this.serializableProperties`.